dataprofiler/profilers/helpers/report_helpers.py

```diff
@@ -41,37 +41,105 @@ def flat_dict(od, separator='_', key=''):
            } if isinstance(od, dict) else {key:od}
 
 
-def _prepare_report(report, output_format=None):
-    if output_format:
+def _prepare_report(report, output_format=None, omit_keys=[]):
+    """
+    Prepares report dictionary for users upon request.
+
+    :param report: contains the values identified from the profile
+    :type report: dict()
+    :param output_format: designation for how to format the returned report
+    :type output_format: dict()
+    :param omit_keys: Keys to omit from the output report, to omit keys in the 
+                      report a '.' represents a level of recursion example: 
+                      report: { 'test1': { 'test2': val, 'test3': val }, 
+                      to omit key 'test3' from report: omit_keys=['test1.test3']
+                      wildcards are also possible, so: omit_keys=['*.test3']
+    :type omit_keys: list(str)
+    :return report: handle to the updated report
+    :type report: dict()
+    """
+    
+    format_options = ['pretty', 'serializable', 'flat', 'compact']    
```

Comment:
should we bring it outside of the function?

Reply:
Also this list isn't accessed at all within the function, what is its purpose exactly?

Reply:
Fair point @ChrisWallace2020, I removed the requirement - I'll add to the description. 

@AnhTruong These are just the modifications that can occur in the function. The function itself used to check if one of those options existed.